app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts {
+  &.reply {
+    @include forum-post;
```

Comment:
Not really. Why does a reply need a post's styles? There's no post in the reply page?